components/StyledBody.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function StyledBody(props) {
```

Comment:
I personally find this art of use of "Styled" confusing. Normally you should write :
```suggestion
export default function Body(props) {
return <StyledBody> etcetera...

than const StyledBody = styled.div`
things...
```